examples/Viewer.tsx

```diff
@@ -69,6 +77,27 @@ interface ViewerState {
     timeStep: number;
     totalDuration: number;
     uiDisplayData: UIDisplayData;
+    filePending: {
+        type: string;
+        trajectory: string;
+        template: { [key: string]: any };
+        templateData: { [key: string]: any };
+    };
+}
+
+interface BaseType {
+    isBaseType: true;
+    id: string;
+    data: string;
+    match: string;
+}
+
+interface CustomType {
+    [key: string]: {
+        "python::module": string;
+        "python::object": string;
+        parameters: { [key: string]: any };
```

Comment:
Do we really know more about parameters than this?  It would be cool if you could get to something like this, for example: `{ [key:string] : BaseType | CustomType}`